docker/utils/prepare-local-wordpress-database.rb

```diff
@@ -17,20 +19,40 @@ def main
     display_name: 'regional'
   )
 
-  run_sql_to_fix_images()
+  display_substitution_counts("Starting count of required changes:")
+  run_sql_to_fix_images_and_links()
+  display_substitution_counts("Ending count of required changes (all should be 0):")
 end
 
-def run_sql_to_fix_images
-  domain = 'http://intranet.docker'
+def display_substitution_counts(message)
+  puts "#################################################"
+  puts message
+  sql = <<~SQL
+  SELECT COUNT(*) AS wp_options_changes FROM wp_options WHERE option_value LIKE "%https://s3-eu-west-1.amazonaws.com/moj-wp-prod/wp-content/%" AND option_name LIKE "%need_to_know_image%";
+  SELECT COUNT(*) AS wp_posts_guid_changes FROM wp_posts WHERE guid LIKE "%https://intranet.justice.gov.uk/wp-content/%";
+  SELECT COUNT(*) AS wp_posts_post_content_relative_link_changes FROM wp_posts WHERE post_content LIKE "%https://intranet.justice.gov.uk/%";
+  SELECT COUNT(*) AS wp_posts_post_content_remove_s3_uri FROM wp_posts WHERE post_content LIKE "%https://s3-eu-west-1.amazonaws.com/moj-wp-prod/wp-content%";
+  SELECT COUNT(*) AS wp_postmeta_meta_value_relative_link_changes FROM wp_postmeta WHERE meta_value LIKE "%https://intranet.justice.gov.uk%";
+  SQL
+
+  puts `mysql -h${DB_HOST} -u${DB_USER} -p${DB_PASSWORD} ${DB_NAME} --table -e '#{sql.gsub("\n", " ")}'`
```

Comment:
Yes, that way I don't have to do anything special/rubyish to get the counts back out of the database.  It's essentially a go/no-go gauge for 'did the substitution sql work properly?'